### PR TITLE
Add some notes about content and stretchX/Y in sprite JSON files

### PR DESCRIPTION
**Pull request by kkaefer**

### CODE REVIEW

File: docs/pages/style-spec.js [hunk 1]
```diff
@@ -1053,14 +1055,51 @@ export default class extends React.Component {
...
-                                vector tile features with a <code>icon</code>{' '}
-                                property with the value <code>poi</code>.
+                                <p>
+                                    Then the style could refer to this sprite
+                                    image by creating a symbol layer with the
+                                    layout property
+                                    <code>"icon-image": "poi"</code>, or with
+                                    the tokenized value{' '}
+                                    <code>"icon-image": "{`{icon}`}"</code> and
+                                    vector tile features with a{' '}
```
Comment (katydecorah) [inline docs/pages/style-spec.js]:
```suggestion
                                    vector tile features with an{' '}
```

File: docs/pages/style-spec.js [hunk 2]
```diff
@@ -1053,14 +1055,51 @@ export default class extends React.Component {
...
+                                    <code>icon</code> property with the value{' '}
+                                    <code>poi</code>.
+                                </p>
+                                <p>
+                                    Apart from the required <code>width</code>,
+                                    <code>height</code>, <code>x</code>, and{' '}
+                                    <code>y</code>
```
Comment (katydecorah) [inline docs/pages/style-spec.js]: Do we need a space here?

```suggestion
                                    <code>y</code>{' '}
```

image

File: docs/pages/style-spec.js [hunk 3]
```diff
@@ -1053,14 +1055,51 @@ export default class extends React.Component {
...
+                                    layout property
+                                    <code>"icon-image": "poi"</code>, or with
+                                    the tokenized value{' '}
+                                    <code>"icon-image": "{`{icon}`}"</code> and
+                                    vector tile features with a{' '}
+                                    <code>icon</code> property with the value{' '}
+                                    <code>poi</code>.
+                                </p>
+                                <p>
+                                    Apart from the required <code>width</code>,
```
Comment (katydecorah) [inline docs/pages/style-spec.js]: This is minor, but maybe another space here?
```suggestion
                                    Apart from the required <code>width</code>,{' '}
```

File: docs/pages/style-spec.js [hunk 4]
```diff
@@ -1053,14 +1055,51 @@ export default class extends React.Component {
...
+                                    properties, the following optional
+                                    properties are supported:
+                                </p>
+                                <ul>
+                                    <li>
+                                        <code>content</code>: An array of four
+                                        numbers, with the first two specifying
+                                        the left, top corner, and the last two
+                                        specifying the right, bottom corner. If
+                                        present, and if the icon uses
```
Comment (katydecorah) [inline docs/pages/style-spec.js]: Another space

```suggestion
                                        present, and if the icon uses{' '}
```

File: docs/pages/style-spec.js [hunk 5]
```diff
@@ -1053,14 +1055,51 @@ export default class extends React.Component {
...
+                                            <code>icon-text-fit</code>
+                                        </a>
+                                        , the symbol's text will be fit inside
+                                        the content box.
+                                    </li>
+                                    <li>
+                                        <code>stretchX</code>: An array of
+                                        two-element arrays, consisting of two
+                                        numbers that represent the <em>from</em>{' '}
+                                        position and the
```
Comment (katydecorah) [inline docs/pages/style-spec.js]: Another space:
```suggestion
                                        position and the{' '}
```

**Comment by kkaefer** [comment]
Thanks for the review! Why do we manually need to add spaces? I was under the impression that a linebreak + indentation would be sufficient?

**Comment by katydecorah** [comment]
@kkaefer all the missing spaces appear to come between a string and an HTML element. I tested and on commit Prettier will indent and add spaces between a string and HTML element. I'm not exactly sure why these spaces were missing from the initial commit.

**Comment by kkaefer** [comment]
I originally had formatted it differently, but the formatting was changed during the commit.

**Pull request merged**